src/content/reference/faqs.mdx

```diff
@@ -53,36 +53,7 @@ Please see our [ArgoCD Configuration Guide](self-hosted/manage/argocd.mdx) for o
 
 ## How to use private images?
 
-In order to use your private registry credentials you must configure a [pull secret](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/) and use it in your Kubernetes manifests.
```

Comment:
This is a very old explanation, we now have Registry Credentials supported at the Okteto Platform level